docs/branching-strategy.md

```diff
@@ -0,0 +1,29 @@
+# Braching strategy
+
+This project currently uses a simple branching strategy that is inspired by [git-flow](http://nvie.com/posts/a-successful-git-branching-model/). _Diet Git Glow_, if you will.
+
+## `develop` branch
+
+The most recently delivered development changes reside in `develop`.
+
+This has been configured as a restricted branch on GitHub and only project maintainers are able to push to this branch (or merge pull requests into it).
```

Comment:
I think you need to add how the deployment to staging would work and add the link to the staging environment too